File: example02threading/src/main/java/foo/bar/example/forethreading/ui/CounterFragment.java
package foo.bar.example.forethreading.ui;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import foo.bar.example.forethreading.R;


public class CounterFragment extends Fragment {

    public static CounterFragment newInstance() {
        CounterFragment fragment = new CounterFragment();
        return fragment;
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_counter, null);
    }

}

File: example-kt-02coroutine/src/main/java/foo/bar/example/forecoroutine/ui/CounterFragment.kt
package foo.bar.example.forecoroutine.ui

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import androidx.fragment.app.Fragment
import foo.bar.example.forecoroutine.R

/**
 * Copyright © 2019 early.co. All rights reserved.
 */
class CounterFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_counter, null)
    }

    companion object {

        fun newInstance(): CounterFragment {
            return CounterFragment()
        }
    }

}
